Composer/packages/client/src/pages/home/styles.ts

```diff
@@ -234,19 +231,26 @@ export const mediaCardItem = {
   imageCover: css`
     ${cardItem.imageCover}
     width: 100%;
+    margin-bottom: 12px;
+    overflow: hidden;
+    display: flex;
     min-height: 95px;
     height: calc(17vw * 95 / 244);
```

Comment:
What is this calculation mean. How did you arrive at it?

Reply:
17vw is image current width. it mean `ImageCurrentWidth / ImageCurrentHeight` should equal to origin aspect ratio `95 / 244`. I wrote this to implement the image can also increase height and keep ratio.

Reply:
Can u create variables for those constants and use them here.